### PR TITLE
fix: store markdown string instead of MDAST in MDX field

### PR DESCRIPTION
**Pull request by schettn**
ref: https://github.com/jaenjs/jaen/issues/9

### CODE REVIEW

File: my-gatsby-site/src/pages/mdx.tsx [hunk 1]
```diff
@@ -25,18 +25,37 @@ const Page: React.FC<PageProps> = ({location, pageContext}) => {
...
+  //         setValue(value)
+  //       }}
+  //       isEditing={true}
+  //     />
+
+  //     <button onClick={save} disabled={!needsSaving}>
+  //       Save
+  //     </button>
+  //   </>
+  // )
```
Comment (kleberbaum) [inline my-gatsby-site/src/pages/mdx.tsx:58]:
```suggestion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Looks good, just a few tweaks needed.

File: my-gatsby-site/src/pages/mdx.tsx (hunk 1 above)
Comment (schettn) [reply my-gatsby-site/src/pages/mdx.tsx:58]: No, this in on purpose

Comment (schettn) [reply my-gatsby-site/src/pages/mdx.tsx:58]: Its dummy code in the example repo

**Comment by schettn** [comment]
Cannot merge as this could kill the image feature for experiments

**Pull request merged**